sys/posix/pthread/include/pthread_once.h

```diff
@@ -24,14 +24,17 @@ extern "C" {
 /**
  * @brief           Datatype to supply to pthread_once().
  */
-typedef volatile int pthread_once_t;
```

Comment:
Not really clear to me why `pthread_once_t` was declared to be `volatile`, propably to vaoid optimizing it out. But I don't see any reason why it shouldn't be optimized out.